public/speed-testJS.js

```diff
@@ -584,8 +584,11 @@
                 myChart.setOption(option, true);
         }
         var baseUrl = (version === 'IPv6') ? 'http://' + testPlan.baseUrlIPv6 : 'http://' + testPlan.baseUrlIPv4;
-
-        var uploadHttpConcurrentTestSuite = new window.uploadHttpConcurrentProgress(baseUrl + '/upload', 'POST', 1, 15000, 15000, 2, uploadHttpOnComplete, uploadHttpOnProgress,
+        var concurrentRuns = 1;
+        if(uploadSize>= 10671896){
+            concurrentRuns=6
+        }
+        var uploadHttpConcurrentTestSuite = new window.uploadHttpConcurrentProgress(baseUrl + '/upload', 'POST', concurrentRuns, 20000, 20000, 10, uploadHttpOnComplete, uploadHttpOnProgress,
```

Comment:
The same with the call here for 20000 20000 10. Ideally put them at one place so that any changes are automatically reflected here and in the examples uploadApp.js